src/main/java/org/folio/inventory/storage/external/ExternalStorageModuleHoldingCollection.java

```diff
@@ -1,22 +1,19 @@
 package org.folio.inventory.storage.external;
 
-import io.vertx.core.Vertx;
-import io.vertx.core.http.HttpClient;
-import io.vertx.core.json.JsonObject;
+import java.util.UUID;
+
 import org.folio.inventory.domain.Holding;
 import org.folio.inventory.domain.HoldingCollection;
 
-import java.util.UUID;
+import io.vertx.core.http.HttpClient;
+import io.vertx.core.json.JsonObject;
 
 class ExternalStorageModuleHoldingCollection
   extends ExternalStorageModuleCollection<Holding>
   implements HoldingCollection {
 
-  ExternalStorageModuleHoldingCollection(Vertx vertx,
-                                         String baseAddress,
-                                         String tenant,
-                                         String token,
-                                         HttpClient client) {
+  ExternalStorageModuleHoldingCollection(String baseAddress, String tenant,
+    String token, HttpClient client) {
```

Comment:
the line breaks here seem strange.  I actually prefer the old formatting, as it makes the arguments easier to see.

Reply:
What do you mean by strange?

I used to prefer the old formatting too (except for the horizontal alignment). However many folks don't and it takes up a significant amount of vertical space.

Reply:
What would be your ideal formatting for these parameters?

Reply:
If we can't stack them, could they be on their own line?

Reply:
I don't think this is an important enough issue to hold up the PR.  I get your point about the fact that we have several different approaches to formatting arguments.

Reply:
> I don't think this is an important enough issue to hold up the PR. 

I wasn't holding it up.

> I get your point about the fact that we have several different approaches to formatting arguments.

It's a long standing challenge that hasn't really been addressed (probably in part because my own opinion has varied over time)